src/azure-cli/azure/cli/command_modules/batch/_help.py

```diff
@@ -74,6 +74,103 @@
 long-summary: List the endpoints that a Batch Compute Node under this Batch Account may call as part of Batch service administration. If you are deploying a Pool inside of a virtual network that you specify, you must make sure your network allows outbound access to these endpoints. Failure to allow access to these endpoints may cause Batch to mark the affected nodes as unusable. For more information about creating a pool inside of a virtual network, see https://docs.microsoft.com/azure/batch/batch-virtual-network."
 """
 
+helps['batch account identity'] = """
+type: group
+short-summary: Manage identities of a batch account.
+"""
+
+helps['batch account identity assign'] = """
+type: command
+short-summary: Add managed identities to an existing batch account.
+examples:
+  - name: Add a system assigned managed identity to an existing batch account.
+    text: >
+        az batch account identity assign --name MyBatchAccount --resource-group MyResourceGroup --system-assigned
+  - name: Add a user assigned managed identity to an existing batch account.
+    text: >
+        az batch account identity assign --name MyBatchAccount --resource-group MyResourceGroup --user-assigned MyAssignedId
+"""
+
+helps['batch account identity remove'] = """
+type: command
+short-summary: Remove managed identities from an existing batch account.
+examples:
+  - name: Remove a system assigned managed identity from an existing batch account.
+    text: >
+        az batch account identity remove --name MyBatchAccount --resource-group MyResourceGroup --system-assigned
+  - name: Remove a user assigned managed identity from an existing batch account.
+    text: >
+        az batch account identity remove --name MyBatchAccount --resource-group MyResourceGroup --user-assigned MyAssignedId
+  - name: Remove all user assigned managed identities from an existing batch account.
+    text: >
+        az batch account identity remove --name MyBatchAccount --resource-group MyResourceGroup --user-assigned MyAssignedId
```

Comment:
> For the convenience scenario to remove all user assigned identities, --user-assigned with no values should remove all user assigned identities with proper warnings.
```suggestion
  - name: Remove all user assigned managed identities from an existing batch account.
    text: >
        az batch account identity remove --name MyBatchAccount --resource-group MyResourceGroup --user-assigned
```

Reply:
added the ability to take in an empty value and delete all